app/navigation/Navigator.scala

```diff
@@ -36,6 +36,8 @@ class Navigator @Inject() () {
     case HasGoodsDescriptionPage => navigateFromHasGoodsDescription
     case GoodsDescriptionPage    => _ => routes.CountryOfOriginController.onPageLoad(NormalMode)
     case CommodityCodePage       => _ => routes.HasCorrectGoodsController.onPageLoad(NormalMode)
+    case CountryOfOriginPage     => _ => routes.CommodityCodeController.onPageLoad(NormalMode)
```

Comment:
Can we have these rows in the order of the pages (so one lead on from the other so it looks nicer)

Reply:
Done it :D